### PR TITLE
Closes #187 Update vignettes: add BICR alongside IRF

### LINKED ISSUE #187: Update vignettes: adrs.Rmd & adtte.Rmd to include the acronym BICR alongside IRF
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

BICR=Blinded Independent Central Review which is industry used acronym.  
Need to update the documentation in Admiralonco to add '/Blinded Independent Central Review (BICR)' where we say 'Independent Review Facility (IRF)'  -

### PR DESCRIPTION
**Pull request by kaz462**
Please add "Closes #<issue number>" to the title of the pull request. Then the
issue is closed automatically once it is merged to `main`.

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralonco/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

The following line requests to update the man pages by the "Man Pages" workflow.

/roxygenize

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@kaz462 thanks for another contribution! Heads up for future that you dont need to add the PR to the project board on the right hand side, we only do this for the issues (and each of them get linked to the PR anyway).

@HinalPat i added you as reviewer but feel free to delegate.

**Comment by rossfarrugia** [comment]
p.s. if the spellcheck auto-check fails it will likely be because you just need to add `BICR` to `inst/WORDLIST` file. Sometimes it oddly let's things like this through though!

**Comment by kaz462** [comment]
@HinalPat Thanks for assigning me the issue!
@rossfarrugia Good to know that we only need issues in the project board, no need to add PR. 
Thanks for your information about the spellcheck and potential words to be added in `inst/WORDLIST` if it fails - good learnings for me!

File: vignettes/adrs.Rmd [hunk 1]
```diff
@@ -576,7 +576,7 @@ dataset_vignette(
 )
 ```
 
-## Derive Parameters using Independent Review Facility (IRF) responses {#irf}
+## Derive Parameters using Independent Review Facility (IRF)/Blinded Independent Central Review (BICR) responses {#irf}
 
 All of the above steps can be repeated for different sets of records, such as now using assessments from the Independent
 Review Facility instead of investigator. For this you would just need to replace the first steps with selecting the required
```
Comment (bundfussr) [inline vignettes/adrs.Rmd:582]: I would replace "the Independent Review Facility" with "IRF/BICR".

**Review by bundfussr — CHANGES_REQUESTED** [review]
Thanks @kaz462 , looks almost good to me.
Only one place where "BICR" should be added.
If you add it and fix the merge conflicts, we can merge the PR.

**Review by bundfussr — APPROVED** [review]

**Pull request merged**